server/src/main/java/org/apache/iotdb/db/qp/executor/QueryProcessExecutor.java

```diff
@@ -130,13 +130,16 @@ public boolean processNonQuery(PhysicalPlan plan) throws ProcessorException {
       case SET_STORAGE_GROUP:
         return setStorageGroup((SetStorageGroupPlan) plan);
       case DELETE_STORAGE_GROUP:
-        return deleteStorageGroup((DeleteStorageGroupPlan) plan);  
+        return deleteStorageGroup((DeleteStorageGroupPlan) plan);
       case PROPERTY:
         PropertyPlan property = (PropertyPlan) plan;
         return operateProperty(property);
       case TTL:
         operateTTL((SetTTLPlan) plan);
         return true;
+      case LOAD_CONFIGURATION:
+        IoTDBDescriptor.getInstance().loadHotModifiedProps();
```

Comment:
if some param is error ,some param is true , the load is dirty.
for example:
default pararm is :
`
tsfile_size_threshold = 1000
group_size_in_byte = 300
`
i will change it to 
`
tsfile_size_threshold = 100
group_size_in_byte = 30**f**
`
it look's like tsfile_size_threshold will be start work

Reply:
These parameters need to be updated by the user, so I will update all parameters until I encounter a wrong parameter. In the scenario you described, is there any problem? Equivalent to that the user only updated `tsfile_size_threshold = 100`.

Reply:
write  to disk size will all allways max than `tsfile_size_threshold`

Reply:
I don't think so, `tsfile_size_threshold` is just a lower bound to close the tsfile. So if `tsfile_size_threshold` is smaller than `group_size_in_byte`, it just means you can close the tsfile after you flush one memtable, which is acceptable in IoTDB and will not crash the system.

Reply:
yes, i haven't thought of a scene that could cause a crash, but if there are multiple interrelated parameters， may be have some hidden bugs.